### ISSUE TITLE
host performance stats somewhere

### ISSUE THREAD

**Issue by garfieldnate** [SoarGroup/Soar#459]
The readme has a prominent section on Soar performance, but all of the links are dead. We don't have the old Jenkins anymore and we don't store the performance numbers in the repository. We do run the performance tests in CI, though.

* [ ] Upload performance results along with sha1/tag information to another repo; could be either Soar website repo or a new repo dedicated to performance.
* [ ] Put the raw numbers in a CSV
* [ ] From the CSV, generate performance graphs over time for the different stats. The X-axis can have either dates or sha-1's. The tags (9.6.2, 9.6.3, etc.) should be marked prominently.
* [ ] MAYBE: If using a new repository, create a website for the repo so that we have a direct display URL.
* [ ] Update the URLs in the readme of this repo to point to the new CSV and graph

**Comment by ShadowJonathan** [comment]
Note: GitHub actions only stores CI artifacts for 90 days, so if you want to have the files for the last 90 days, I suggest grabbing them quickly

**Comment by garfieldnate** [comment]
> Note: GitHub actions only stores CI artifacts for 90 days, so if you want to have the files for the last 90 days, I suggest grabbing them quickly

Yeah, most of that data is looong gone by now XD I'm not worried about it for now.
